drop-boxes/register-nanopore-dropbox/register-nanopore.py

```diff
@@ -137,6 +137,7 @@ def createExperimentFromMeasurement(transaction, currentPath, space, project, me
     # runExperiment.setPropertyValue("Q_EXTERNALDB_ID",) best skip and parse sample information at sample level, no experiment-wide ID from what I can tell
     # handle measured samples
     for barcode in rawDataPerSample.keySet():
+        print "handling barcode: "+barcode
```

Comment:
Small suggestion
```suggestion
        print "Registering Nanopore measurements for sample with code: "+barcode
```

Reply:
+1

Reply:
removed the print in the latest version. do you think it would be heplful?

Reply:
I would say either leave it out or change it as suggested by @sven1103 
I do not think this is necessary but this is mainly because you should not have to look at the log file except something goes wrong and then I expect the information on what went wrong and when it did fail to be in the error message.